### ISSUE TITLE
Hero Banner - Pixelated/Low quality 

### ISSUE THREAD

**Issue by colinvlasak** [aemsites/stericycle-shared#211]
Hero banner images pixelated/low quality – this is across the site. Also appears to be scaled larger or zoomed in.

Can we update this site wide?

o	EDS (blurry and big)
![image](https://github.com/user-attachments/assets/cbcacfa0-708f-4a1d-a615-b9a867414504)

•	Live Site (crisp and smaller)
 
![image](https://github.com/user-attachments/assets/f5d620ea-184f-4093-81e6-9c42c10cf9b9)

**Comment by kronnox** [comment]
@jindaliiita I assume this can be closed?

**Comment by jindaliiita** [comment]
Closing this as this is covered via https://github.com/aemsites/stericycle-shared/pull/391

**Comment by colinvlasak** [comment]
@jindaliiita I just added a new Hero banner to a new page and the image is very blurry and pixelated. Can you confirm how this can be fixed not only for this page but for all pages? I want to ensure every time we add a new image we don't have this issue

**Comment by jindaliiita** [comment]
It seems that the optimized image being returned from the server is causing issues.

Image without optimization: [Link](https://main--shredit--stericycle.aem.page/en-us/media_1dad5b4cd161db5200f50b892eda8760c63f9018a.png)
Image with optimization: [Link](https://main--shredit--stericycle.aem.page/en-us/media_1dad5b4cd161db5200f50b892eda8760c63f9018a.png?width=2000&format=webply&optimize=medium)

@bstopp @kronnox @dkuntze @MarquiseRosier Do you have any suggestions on how we can address this issue? One approach I can think of is calling createOptimizedPicture with more breakpoints specifically for the hero block.

**Comment by jindaliiita** [comment]
> It seems that the optimized image being returned from the server is causing issues.
> 
> Image without optimization: [Link](https://main--shredit--stericycle.aem.page/en-us/media_1dad5b4cd161db5200f50b892eda8760c63f9018a.png) Image with optimization: [Link](https://main--shredit--stericycle.aem.page/en-us/media_1dad5b4cd161db5200f50b892eda8760c63f9018a.png?width=2000&format=webply&optimize=medium)
> 
> @bstopp @kronnox @dkuntze @MarquiseRosier Do you have any suggestions on how we can address this issue? One approach I can think of is calling createOptimizedPicture with more breakpoints specifically for the hero block.

Reiterating once again, @bstopp @kronnox @MarquiseRosier, could you please check the comment above? ⬆️

**Comment by colinvlasak** [comment]
Just wanted to note this is a problem for all hero banners across the site and need to figure out a fix.

**Comment by jindaliiita** [comment]
@colinvlasak
With this [PR](https://github.com/aemsites/stericycle-shared/pull/696), the image quality has improved slightly. Could you please take a look? I believe we cannot further improve the quality because we are currently fetching an optimized image from EDS. If we remove the optimization, larger image sizes will be fetched, which could significantly increase the page load time.

Let me know if this works for you!
Sample pages
Before:
https://main--shredit--stericycle.aem.live/en-us/national-accounts/una
After:
https://issue-211--shredit--stericycle.aem.live/en-us/national-accounts/una

Before:
https://main--shredit--stericycle.aem.live/drafts/piyush/en-us
After:
https://issue-211--shredit--stericycle.aem.live/drafts/piyush/en-us

**Comment by kronnox** [comment]
I think it already looks a lot better. At least for my screen resolution 👍🏻

**Comment by colinvlasak** [comment]
@jindaliiita image quality does look a bit better. Is it just scaling the size down though? The height appears smaller. I am concerned this may cause some issues if so. A lot of the banners are built to be taller and fill more space. The edges may end up looking odd in some instances.

**Comment by jindaliiita** [comment]
> @jindaliiita image quality does look a bit better. Is it just scaling the size down though? The height appears smaller. I am concerned this may cause some issues if so. A lot of the banners are built to be taller and fill more space. The edges may end up looking odd in some instances.

No, we have neither reduced nor increased the height. We have just added some additional custom breakpoints for image optimization.
Let us know if you have any example page. Otherwise we can close this.

**Comment by colinvlasak** [comment]
@jindaliiita this one is a good example. It looks good as is but it the fix were to make it zoom out it gets wonky

https://main--shredit--stericycle.aem.live/en-us/secure-shredding-services/shredding-events

![image](https://github.com/user-attachments/assets/064bbcb8-0d48-4bd3-a3d8-2bb68abe8a88)


![image](https://github.com/user-attachments/assets/17c35764-4b20-4ac4-b338-f2ac69866732)

**Comment by jindaliiita** [comment]
> @jindaliiita this one is a good example. It looks good as is but it the fix were to make it zoom out it gets wonky
> 
> https://main--shredit--stericycle.aem.live/en-us/secure-shredding-services/shredding-events
> 
> ![image](https://private-user-images.githubusercontent.com/178829136/401566617-064bbcb8-0d48-4bd3-a3d8-2bb68abe8a88.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY0MzE2MTQsIm5iZiI6MTczNjQzMTMxNCwicGF0aCI6Ii8xNzg4MjkxMzYvNDAxNTY2NjE3LTA2NGJiY2I4LTBkNDgtNGJkMy1hM2Q4LTJiYjY4YWJlOGE4OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwOVQxNDAxNTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMDE2M2E0YThkYzNjMDU1NmQ2ZDMzOTJiOWQ1MWQ5OTkyNzQxMGIwZmY5MGMwMzFiOGZiNTE5YTk1ZmE2YjM5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.rHr7wF6GNPapH48J3kaN0CeqNHMtGqSkInbvWYNZO5o)
> 
> ![image](https://private-user-images.githubusercontent.com/178829136/401566756-17c35764-4b20-4ac4-b338-f2ac69866732.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY0MzE2MTQsIm5iZiI6MTczNjQzMTMxNCwicGF0aCI6Ii8xNzg4MjkxMzYvNDAxNTY2NzU2LTE3YzM1NzY0LTRiMjAtNGFjNC1iMzM4LWYyYWM2OTg2NjczMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwOVQxNDAxNTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNWVkODE3ZTA5ZGFlYTMzZTNlOWNhMzRkYjE1MTBhMGIzMTlkNWJlYmM5NDNiMWRmNzc4NGVjMWU3YzlkMDEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.6BNdgm0CI13moqJVPWVi1K1fG2bGnG0DOqNl7YAHnsg)

@colinvlasak The image quality fix is not affecting this. This is a separate issue where, ideally, the text on the left side should occupy less space or width.

**Comment by colinvlasak** [comment]
@jindaliiita I was referring to the way the truck stretches out if we show too much of the image. I circled this with a 1. With regards to the text wrapping is there anything we can do there? It's not a showstopper but doesn't look great. 

![image](https://github.com/user-attachments/assets/c155ef99-9be3-4d31-b4f9-27794ae72e77)

**Comment by jindaliiita** [comment]
<img width="1728" alt="Screenshot 2025-01-09 at 7 57 18 PM" src="https://github.com/user-attachments/assets/c81c0a06-4a2b-4f6a-9b82-c1ff1515aee3" />

The same behaviour is present on the live site as well. I guess it's not related to the image, but rather because you are viewing the page with a larger viewport than your screen.

**Comment by jindaliiita** [comment]
Closing this issue as per the discussion.
